frontend/pages/news/requests/index.tsx

```diff
@@ -0,0 +1,37 @@
+import { Stack } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import Link from '~/components/Link';
+import ArticleRequest from '~/components/News/ArticleRequest';
+import genGetProps from '~/functions/genGetServerSideProps';
+import { useArticleRequestsQuery } from '~/generated/graphql';
+import routes from '~/routes';
+
+export default function ArticleRequests() {
```

Comment:
What do you think about putting requests and rejections one one page? I'm thinking 2 column desktop layout, stack on mobile.